Modules/_ssl.c

```diff
@@ -5626,12 +5624,25 @@ _ssl_enum_certificates_impl(PyObject *module, const char *store_name)
     PCCERT_CONTEXT pCertCtx = NULL;
     PyObject *keyusage = NULL, *cert = NULL, *enc = NULL, *tup = NULL;
     PyObject *result = NULL;
+    static DWORD system_stores[] = {
+        CERT_SYSTEM_STORE_LOCAL_MACHINE,
+        CERT_SYSTEM_STORE_LOCAL_MACHINE_ENTERPRISE,
+        CERT_SYSTEM_STORE_LOCAL_MACHINE_GROUP_POLICY,
+        CERT_SYSTEM_STORE_CURRENT_USER,
+        CERT_SYSTEM_STORE_CURRENT_USER_GROUP_POLICY,
+        CERT_SYSTEM_STORE_SERVICES,
+        CERT_SYSTEM_STORE_USERS};
+    size_t i, system_stores_count = sizeof(system_stores) / sizeof(DWORD);
```

Comment:
This style is confusing and hard to read. Split ``size_t i`` and ``size_t system_stores_count`` into two lines.